includes/class-wpsms-gateway.php

```diff
@@ -971,7 +971,8 @@ public static function help()
         $help     = $sms->help;
         $document = isset($sms->documentUrl) ? $sms->documentUrl : false;
 
-        return $document ? sprintf(esc_html__('%s <a href="%s" target="_blank">Documentation</a>', 'wp-sms'), $help, $document) : $help;
+        // translators: %1$s: Helpful tip, %2$s: Gateway documentation URL
+        return $document ? sprintf(esc_html__('%1$s <a href="%2$s" target="_blank">Documentation</a>', 'wp-sms'), $help, $document) : $help;
```

Comment:
I think this probably should change to `__` @foadadeli 

Since is only this one, I'll fix it after the merge.

Reply:
@mostafasoufi  Thank you!